CMakeLists.txt

```diff
@@ -462,6 +462,19 @@ else()
 	RemoveFlag(CMAKE_C_FLAGS_MINSIZEREL CACHE "/DNDEBUG" "Default C flags for MinSizeRel")
 endif()
 
+if(ENABLE_MYSQL)
+	execute_process(COMMAND mysql_config --cflags OUTPUT_VARIABLE MYSQL_CFLAGS OUTPUT_STRIP_TRAILING_WHITESPACE)
+	string(REGEX REPLACE "-I" "" MYSQL_CFLAGS "${MYSQL_CFLAGS}")
+	string(REGEX REPLACE "-DNDEBUG" "" MYSQL_CFLAGS "${MYSQL_CFLAGS}")
+	execute_process(COMMAND mysql_config --libs OUTPUT_VARIABLE MYSQL_LIBS OUTPUT_STRIP_TRAILING_WHITESPACE)
+	add_subdirectory("src/modules/mariadbpp/")
+
+	# ignore this warning for mariadbpp submodule
+	if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+		add_compile_options(-Wshorten-64-to-32)
```

Comment:
The comments says "ignore", should there be a `no-` somewhere in the added option?

Reply:
No, the comment is actually correct - the warning is not applied to the mariadbpp submodule since it comes before the `add_subdirectory("src/modules/mariadbpp/")` command.

Reply:
Please would you change the comment to make that more obvious?

Reply:
Updated.